### PR TITLE
#386 CreateView for schema conditions

### PR DESCRIPTION
**Pull request by vryazanov**
#386 
* create view for schema conditions
* tests

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#446](https://github.com/timelesslounge/timelessis/issues/446) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#446](https://github.com/timelesslounge/timelessis/issues/446) is assigned to @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/30ab7bbd-a067-4fb4-8d16-16535416a9d6); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by 0crat** [comment]
Job `gh:timelesslounge/timelessis#446` already assigned to @vladarefiev, can't assign to @vladarefiev

File: tests/integration/it_scheme_types_test.py [hunk 1]
```diff
@@ -31,3 +33,18 @@ def test_scheme_condition_list(client, db_session):
                                     scheme_type_id=scheme_type.id))
     assert conditions.status_code == HTTPStatus.OK
     assert b"Test condition" in conditions.data
+
+
+def test_scheme_condition_create(client, db_session):
+    """ Test that CreateView works correctly and creates an instance """
+    scheme_type = factories.SchemeTypeFactory()
```
Comment (vladarefiev) [inline tests/integration/it_scheme_types_test.py]: @vryazanov according to your test, it's create action. But actually here is not create here, it looks like edit. Is it ok?

Comment (vryazanov) [reply tests/integration/it_scheme_types_test.py]: @vladarefiev test is about creation of `SchemeTypeCondition`, not `SchemeType`.  `scheme_type` is just a helper, it's needed in url path

**Comment by vladarefiev** [comment]
@rultor merge

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 21 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job [#446](https://github.com/timelesslounge/timelessis/issues/446) is not in the agenda of @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), can't inspect

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#446](https://github.com/timelesslounge/timelessis/issues/446) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ypshenychka** [comment]
@vladarefiev According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

No issues were found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by vladarefiev** [comment]
@ypshenychka Confirm

**Comment by ypshenychka** [comment]
@vladarefiev thanks

**Comment by ypshenychka** [comment]
@0crat quality bad

**Comment by 0crat** [comment]
Quality is low, no payment, see [§31](http://www.zerocracy.com/policy.html#31)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**